src/opm/parser/eclipse/share/keywords/900_OPM/D/DRSDTCON

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "DRSDTCON",
+  "sections": [
+    "SCHEDULE"
+  ],
+  "size": 1,
+  "items": [
+    {
+      "name": "DRSDT_MAX",
+      "value_type": "DOUBLE",
+      "dimension": "1"
+    },
+    {
+      "name": "Option",
```

Comment:
This should be capitalized plural: `OPTIONS`

Reply:
OK. I will update the rest of the DRSDTs to make it consistent.